src/rabbit_mgmt_db_handler.erl

```diff
@@ -63,6 +70,7 @@ ensure_statistics_enabled() ->
             ok
     end,
     {ok, StatsLevel} = application:get_env(rabbit, collect_statistics),
+    rabbit_log:info("Using rates mode '~p'~n", [rates_mode()]),
```

Comment:
Also, given that we do not document this "feature", mentioning it in the log messages seems to be a bad idea to me.